client/src/pages/Login.js

```diff
@@ -4,13 +4,19 @@ const Login = () => {
     const [username, setUsername] = useState("");
     const [password, setPassword] = useState("");
     const [passwordconf, setPasswordConf] = useState("");
+    const [firstname, setfirstName] = useState("");
```

Comment:
Super nitpick but maybe make it all camel case like [firstName, setFirstName] and [lastName, setLastName]